README.md

```diff
@@ -4,39 +4,32 @@
 [![CircleCI](https://circleci.com/gh/Azure/aks-engine/tree/master.svg?style=svg)](https://circleci.com/gh/Azure/aks-engine/tree/master)
 [![GoDoc](https://godoc.org/github.com/Azure/aks-engine?status.svg)](https://godoc.org/github.com/Azure/aks-engine)
 
-## Overview
-
-AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
-
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+AKS-Engine is the easiest way to provision a DIY Kubernetes cluster on Azure.
```

Comment:
> Let's bikeshed this.

Yes, let's :)

You infer correctly. The README was the only place where I really focused on re-wording, making it simpler to get AKS-Engine's point across while pointing to the docs for further information.

Github's UI doesn't really help here, but FYI there was some prior discussion here: https://github.com/Azure/aks-engine/pull/253#discussion_r246252586

For the lazy, the original tagline was "AKS-Engine is the easiest way to provision your own Kubernetes cluster on Azure."

@tariq1890 suggested switching "your own" to "an unmanaged". Here was my response to that:

> Thanks for the feedback! I definitely wanted to simplify the verbiage a bit here. "unmanagable" has an odd english connotation, meaning it can be uncontrollable or chaotic. That's why I decided to go with "your own" instead: it gives a sense of personal ownership, but I agree it doesn't clarify well that AKS-Engine has a clear difference from AKS.

> How about "AKS-Engine is the easiest way to provision a DIY Kubernetes cluster on Azure"? I believe that "DIY" promotes a form of creativity (and gives a sense of "unmanaged") while avoiding the chaotic/uncontrollable emotions from "unmanageable", AND it serves the same purpose (making a clear separation between AKS and AKS-Engine).

> What do you think?


That being said, I really do like the idea of using "self-managed" for the same reasons: it makes that clear distinction between AKS and AKS-Engine, while avoiding the negative connotations of "unmanaged". I agree about your thoughts on "DIY". I'll make the change.

Reply:
done

Reply:
+1 for not using DIY - that's not something customers who are looking for support from us but can't use AKS because of their requirements will like to see and I don't think it reflects the state of the project either. "self-managed" seems more appropriate.